VideoLocker/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -367,4 +375,73 @@ public void orientationChangeTest() {
                 activity.getResources().getConfiguration().orientation);
         assertOrientationSetup(activity);
     }
+
+    /**
+     * Testing creation of various fragments in the {@link CourseUnitNavigationActivity}'s
+     * ViewPager, by supplying its {@link CourseUnitPagerAdapter} with all possible
+     * {@link CourseComponent} models.
+     */
+    @Test
+    public void testUnitFragmentCreation() {
```

Comment:
By the way, the `EnrollmentConfigTests` class that you pointed towards isn't being registered as a test suite because it's a non-static inner class. Both it and the `data()` method need to be declared as static for the tests to run properly. The tests are also all failing or throwing an exception on my config (lahore) 😦